docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -111,12 +111,23 @@ function hs_layouts_preprocess_pattern_spotlight(&$variables) {
  */
 function hs_layouts_preprocess_pattern_accordion(&$variables) {
   $paragraph = $variables['context']->getProperty('entity');
+  if (!($paragraph instanceof ParagraphInterface)) {
+    return;
+  }
+
   if (
-    $paragraph instanceof ParagraphInterface &&
     $paragraph->hasField('field_hs_accord_expcol') &&
     !$paragraph->field_hs_accord_expcol->isEmpty() &&
     $paragraph->get('field_hs_accord_expcol')->getString()
   ) {
     $variables['attributes']->addClass('hb-accordion_toggle-all');
   }
+
+  if (
+    $paragraph->hasField('field_accordion_style') &&
+    !$paragraph->field_accordion_style->isEmpty() &&
+    $paragraph->get('field_accordion_style')->getString() == 'secondary'
```

Comment:
It feels like there should be a way to achieve this with less than 3 conditions, but I can't think of one. Works for me.

Reply:
@pookmish @joegl Yes, `getString()` doesn't return `NULL` but an empty string, which works the same in this case. I removed the `isEmpty()` condition.